### DISCUSSION TITLE
Create Issue Document error 422

### DISCUSSION THREAD

**Discussion by antonio8101** [fattureincloud/api#348] [Q&A] [answered]
Collegandovi al vostro portale di documentazione API reference, se provo a testare la ``Create Issue Document``, modificando il solo campo ``company_id`` e lasciando inalterati gli altri, ottengo sempre un errore 422. 
Come mai?

**Comment by emajo** [comment, marked as answer]
Ciao @antonio8101, solitamente la risposta specifica nel dettaglio dove si trova il problema, l'errore ricevuto è sicuramente dovuto a risorse non esistenti nell'account Fatture in Cloud con cui si sta cercando di eseguire la chiamata, ad esempio immagino vengano passati degli gli id di payment_method/payment_account inesistenti.

Le consiglio comunque di controllare l'errore ritornato nella risposta e eventualmente di postarlo qui.

Grazie,
Emanuele,
